docs/help/contributor/style-guide/README.md

```diff
@@ -0,0 +1,20 @@
+# Knative Style Guide
+
+!!! note "A note to first-time Contributors"
+    If you are a first-time contributor, don't feel like you need to understand this Style Guide front-to-back in order to make a contribution.
+
+    ** Though we would obviously love for you to read our Style Guide, we understand if you just want make your contribution and move on.**
```

Comment:
I don't know why we'd have this instead of encouraging people to properly follow it - this perpetuates the issue we discussed of first time contribs not following guidelines if we tell them they don't have to

Reply:
Let me soften the language here...